README.rst

```diff
@@ -522,11 +522,20 @@ The following flags are accepted:
 | current repository. May be :value:`external`, :value:`static` or :value:`vendored`. See                    |
 | `Dependency resolution`_.                                                                                  |
 +-------------------------------------------------------------------+----------------------------------------+
-| :flag:`-index true|false`                                         | :value:`true`                          |
+| :flag:`-index none|lazy|all`                                      | :value:`true`                          |
```

Comment:
The default is `true` but it indicates the options are `none|lazy|all` 🤷

Reply:
Fixed to `all`. Also updated the dependency resolution notes.